### ISSUE TITLE
example notebook on colab: difference in output between reduced precision and full precision

### ISSUE THREAD

**Issue by flaport** [spinsphotonics/fdtdz#25]
Hi, when running the colab notebook with reduced precision, I'm getting the following plot:

![image](https://github.com/spinsphotonics/fdtdz/assets/15832886/f3e0ad13-59e6-44af-974d-36041648ab57)

However, when running with full precision (`use_reduced_precision=False`), I'm getting this plot:

![image](https://github.com/spinsphotonics/fdtdz/assets/15832886/a5ce52a6-adca-488c-9dbc-16cedc9d704b)

notice the small excitation in the top left corner.

I suppose this is a bug? Maybe an issue with the PML?

**Comment by jlu-spins** [comment]
Yes this is a bug, see https://github.com/spinsphotonics/fdtdz/issues/20
